doc/igprof_linux.md

```diff
@@ -0,0 +1,21 @@
+##Notes for IgProf profiler tool in Linux
+
+Due to various clashes between IgProf and some old Linux versions it is necessary to do some previous configuration in order to run IgProf. It looks like the problem is caused when there is a fork during the execution.
```

Comment:
Hm, my first comment here apparently got lost.  The first sentence is a little strong.  I'd just say that we encountered issues with igprof when cvmfs forks.